Estate/models/estate_property.py

```diff
@@ -0,0 +1,110 @@
+from datetime import timedelta
+
+from odoo import models, fields, api
+from odoo.exceptions import ValidationError
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Real Estate Property"
+    _order = "id desc"
+    name = fields.Char(required=True)
+    description = fields.Text()
+    postcode = fields.Char()
+    create_date = fields.Datetime()
+    expected_price = fields.Float(required=True)
+    selling_price = fields.Float()
+    bedrooms = fields.Integer()
+    living_area = fields.Integer()
+    facades = fields.Integer()
+    garage = fields.Boolean()
+    garden = fields.Boolean()
+    garden_area = fields.Integer()
+    garden_orientation = fields.Selection([("north", "North"),("south", "South"),("east", "East"),("west", "West"),])
+    active = fields.Boolean(default=True)
+    state = fields.Selection([("new", "New"),("offer_received", "Offer Received"),("offer_accepted", "Offer Accepted"),("sold", "Sold"),("canceled", "Canceled")], required=True, copy=False , )
```

Comment:
```suggestion
    state = fields.Selection(
        [
            ("new", "New"),
            ("offer_received", "Offer Received"),
            ("offer_accepted", "Offer Accepted"),
            ("sold", "Sold"),
            ("canceled", "Canceled")
        ], required=True, copy=False)
```
For better readability.